data/net/minecraft/resources/HolderSetCodec.mapping

```diff
@@ -1,8 +1,18 @@
 CLASS net/minecraft/resources/HolderSetCodec
+	METHOD <init> (Lnet/minecraft/resources/ResourceKey;Lcom/mojang/serialization/Codec;Z)V
+		ARG 1 registryKey
+		ARG 2 elementCodec
+		ARG 3 disallowInline
 	METHOD create (Lnet/minecraft/resources/ResourceKey;Lcom/mojang/serialization/Codec;Z)Lcom/mojang/serialization/Codec;
 		ARG 0 registryKey
 		ARG 1 holderCodec
 		ARG 2 disallowInline
+	METHOD decode (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 1 ops
+		ARG 2 value
+	METHOD encode (Lnet/minecraft/core/HolderSet;Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
+		ARG 2 ops
+		ARG 3 value
```

Comment:
Args should be `input`, `ops`, `prefix`, as per the parameters on [`Encoder`](https://github.com/Mojang/DataFixerUpper/blob/6d0d1e6281abd94328b135b53f3f3b592dc9f6cd/src/main/java/com/mojang/serialization/Encoder.java#L11)